extensions/src/ACRE2DistortionTestPlugin/TsCallbacks_pluginEvents.cpp

```diff
@@ -17,11 +17,20 @@
 // Handle a command event
 //
 void ts3plugin_onPluginCommandEvent(uint64 serverConnectionHandlerID, const char* pluginName, const char* pluginCommand) {
+    //LOG("[%s], [%s]", pluginName, pluginCommand);
 
     if (pluginName && pluginCommand) {
         // this we pass to a custom TS3 IServer to handle the damn messages
-        if (strstr(pluginName, "acre2") != NULL && CEngine::getInstance()->getExternalServer() ) {
+        if (strstr(pluginName, "acre2") != NULL && CEngine::getInstance()->getExternalServer()) {
             CEngine::getInstance()->getExternalServer()->handleMessage((unsigned char *)pluginCommand);
         }
     }
-}
\ No newline at end of file
+}
+
+// API Compatibility
+// v23
+void ts3plugin_onPluginCommandEvent_v23(uint64 serverConnectionHandlerID, const char* pluginName, const char* pluginCommand, anyID invokerClientID, const char* invokerName, const char* invokerUniqueIdentity) {
+    LOG("[%s], [%s]", invokerName, invokerUniqueIdentity);
```

Comment:
- Remove.
- Verify invoker data?

Reply:
I read the docs, no thanks. :D

Reply:
invokerData is only there if server version is uptodate.. meh